app/jobs/run_ocr_job.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+class RunOCRJob < ApplicationJob
+  delegate :query_service, to: :metadata_adapter
+
+  def perform(file_set_id)
+    file_set = query_service.find_by(id: Valkyrie::ID.new(file_set_id))
+    change_set = DynamicChangeSet.new(file_set)
+    derivative_service.new(change_set).create_derivatives
+  end
+
+  private
+
+    def derivative_service
```

Comment:
call this method derivative_service_factory? it's sort of confusing given the inconsistency with app/derivative-services/plum_derivative_service.rb#hocr_derivative_service